_translator-files/po/de/Running-a-Server.po

```diff
@@ -192,111 +200,115 @@ msgid ""
 "\t<figcaption>How Unregistered Servers work</figcaption>\n"
 "</figure>\n"
 msgstr ""
+"<figure>\n"
+"\t<img src=\"{% include img/de-screenshots/diagram-unreg-server.inc %}\" loading=\"lazy\" alt=\"Diagramm: der Verbindungen zwischen Clients eines nicht registrierten Jamulus-Servers\">\n"
+"\t<figcaption>Wie unregistrierte/private Server funktionieren</figcaption>\n"
+"</figure>\n"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:70
 msgid "If you are running an unregistered server behind a home internet connection, you might need to enable [port forwarding](#port-forwarding) as described below."
-msgstr ""
+msgstr "Wenn du einen unregistrierten/privaten Server über eine private Internetverbindung betreibst, musst du möglicherweise die [Portweiterleitung](#portweiterleitung) wie unten beschrieben aktivieren."
```

Comment:
We're trying to get away from the term private server. See https://github.com/jamulussoftware/jamulus/discussions/2228

Reply:
The discussion result is not clear to me.
Suggest to keep "as is" until the decision is made.

Reply:
Yes. You can leave it in the German version

Reply:
@ann0see Shall I set to resolved?

Reply:
Yes. Just did that